Tutorial.md

```diff
@@ -11,455 +12,240 @@ Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set s
 
-## Collection Resource
-With single the Gist Resource in place let's define the collection of our Gist Resources:
+```markdown
++ Response 201 (application/json)
 
-    ## Gists Collection [/gists{?since}]
-    Collection of all Gists.
+    + Headers
 
-    The Gist Collection resource has the following attribute:
+            Location: /questions/1
 
-    - total
-
-    In addition it **embeds** *Gist Resources* in the Gist Fox API.
-
-
-    + Model (application/hal+json)
-
-        HAL+JSON representation of Gist Collection Resource. The Gist resources in collections are embedded. Note the embedded Gists resource are incomplete representations of the Gist in question. Use the respective Gist link to retrieve its full representation.
-
-        + Headers
-
-                Link: <http:/api.gistfox.com/gists>;rel="self"
-
-        + Body
+    + Body
 
                 {
-                    "_links": {
-                        "self": { "href": "/gists" }
-                    },
-                    "_embedded": {
-                        "gists": [
-                            {
-                                "_links" : {
-                                    "self": { "href": "/gists/42" }
-                                },
-                                "id": "42",
-                                "created_at": "2014-04-14T02:15:15Z",
-                                "description": "Description of Gist"
-                            }
-                        ]
-                    },
-                    "total": 1
+                    "question": "Favourite programming language?",
+                    "published_at": "2014-11-11T08:40:51.620Z",
+                    "url": "/questions/1",
+                    "choices": [
+                        {
+                            "answer": "Swift",
+                            "vote_url": "/questions/1/choices/1",
+                            "votes": 0
+                        }, {
+                            "answer": "Python",
+                            "vote_url": "/questions/1/choices/2",
+                            "votes": 0
+                        }, {
+                            "answer": "Objective-C",
+                            "vote_url": "/questions/1/choices/3",
+                            "votes": 0
+                        }, {
+                            "answer": "Ruby",
+                            "vote_url": "/questions/1/choices/4",
+                            "votes": 0
+                        }
+                    ]
                 }
+```
 
-    ### List All Gists [GET]
-    + Parameters
-        + since (optional, string) ... Timestamp in ISO 8601 format: `YYYY-MM-DDTHH:MM:SSZ` Only Gists updated at or after this time are returned.
-
-    + Response 200
+The next resource in our group allows you to view the detail of a question directly with an ID of a question.
```

Comment:
This is what bothers me. The resource is not allowing us something. Actions – affordances – are. List action allows us the view the detail. Not the resource itself. A nuance I know but feels important to me. The resource represents a data, an information. So a question is resource. "The next resource is Question ..."